ukvm/ukvm-core.c

```diff
@@ -479,7 +479,7 @@ static void setup_cpuid(int kvm, int vcpufd)
         err(1, "KVM_SET_CPUID2 failed");
 }
 
-
+#if 0
```

Comment:
@ricarkol do you mean the interrupt inject code?  perhaps that should be a different issue/PR?